src/lib/graphqlErrorHandler.js

```diff
@@ -20,6 +20,13 @@ export const shouldReportError = originalError => {
   return true
 }
 
+export const formatGravityError = error => {
+  const errorSplit = error.message.split(" - ")
+  const errorObject =
+    errorSplit && errorSplit.length > 1 ? JSON.parse(errorSplit[1]) : null
```

Comment:
Might make sense to wrap the JSON parsing in a try/catch block and fallback to the unparsed version.